### DISCUSSION TITLE
How to Quietly Parse a File

### DISCUSSION THREAD

**Discussion by ThePerfectComputer** [florianschanda/miss_hit#278] [Q&A]
Right now, running ``parser.parse_file()`` will print out style issues. How can I get the parser to run quietly except if there is an error?

**Comment by florianschanda** [comment]
You need to set some flags in the Message_Handler:
https://github.com/florianschanda/miss_hit/blob/f01fbf9cd43c52fdb8fe6dce162c9772637ec199/miss_hit_core/errors.py#L236-L238

Mainly set show_style to `False` and it should be pretty silent.
